docs/source/daml/reference/interfaces.rst

```diff
@@ -147,11 +152,12 @@ Required Interfaces
      :start-after: -- INTERFACE_TRANSITIVE_REQUIRES_CORRECT_BEGIN
      :end-before: -- INTERFACE_TRANSITIVE_REQUIRES_CORRECT_END
 
-- For a template's implementation of an interface to be valid, all its required
-  interfaces must also be implemented by the template.
+- For a template ``T`` to be a valid ``interface instance`` of an interface
```

Comment:
`requires` is not a thing at the moment. we disabled that in lf 1.15.

Reply:
fair, though I don't want to just delete this section. I'll move it to the bottom and add a `.. warning::` in a follow up PR